README.md

```diff
@@ -111,3 +111,29 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Run:
+```Shell
+tools/setup_env.sh
```

Comment:
A few issues here. 

1. The first I run `setup_env.sh`, `make install` fails because it doesn't have root access. Could fix with `sudo`, though by default this creates a file in /usr/local/bin that non-root users can't read/execute.
2. Re-running `setup_env.sh`, `git clone` fails with `fatal: destination path 'git-secrets' already exists and is not an empty directory.`
3. In trying this out, I noticed that our reference to a pre-publication snapshot of the swagger cli jar has expired.

The tweaks below are blunt blunt, but the result works for me...

```
diff --git a/tools/download_swagger_cli.sh b/tools/download_swagger_cli.sh
index 8f7f547..b8a96e8 100755
--- a/tools/download_swagger_cli.sh
+++ b/tools/download_swagger_cli.sh
@@ -1 +1 @@
-wget https://oss.sonatype.org/content/repositories/snapshots/io/swagger/swagger-codegen-cli/2.3.0-SNAPSHOT/swagger-codegen-cli-2.3.0-20170716.142514-29.jar -O swagger-codegen-cli.jar
+wget https://oss.sonatype.org/content/repositories/snapshots/io/swagger/swagger-codegen-cli/2.3.0-SNAPSHOT/swagger-codegen-cli-2.3.0-20170809.095725-71.jar -O swagger-codegen-cli.jar
diff --git a/tools/setup_env.sh b/tools/setup_env.sh
index 1cc214c..fd13a46 100755
--- a/tools/setup_env.sh
+++ b/tools/setup_env.sh
@@ -16,9 +16,10 @@ if which brew
 then
   brew install git-secrets
 else
+  rm -rf git-secrets
   git clone https://github.com/awslabs/git-secrets.git
   cd git-secrets
-  make install
+  sudo make install && sudo chmod o+rx /usr/local/bin/git-secrets
   cd ..
   rm -rf git-secrets
 fi

```

(Note that as we move our dev env to docker, we may want to move angular CLI installation and swagger codegen CLI installation into docker too.)